sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/CloseHandlesInfo.java

```diff
@@ -22,6 +23,18 @@
      */
     public CloseHandlesInfo(Integer closedHandles) {
         this.closedHandles = closedHandles;
+        this.failedHandles = 0; // What should default be??
```

Comment:
Given that this class is generally only used by internal code but needs to be public to support it being returned I think having `failedHandles` being instantiated to 0 is a safe choice, it'll just need to be documented.

Reply:
Added a Javadoc detailing this